src/networking/controllers/todos.ts

```diff
@@ -1,20 +1,52 @@
-import { Todo } from '../../types/globalTypes'
+import { ItemCheck } from '../../features/todosState'
+import { NewItem, Todo } from '../../types/globalTypes'
 import { httpService } from '../httpService'
-interface TodoResponse {
-  userId: number
-  id: number
+export interface TodoResponse {
+  description: string
+  id: string
   title: string
   completed: boolean
 }
 
+const server: string = '/tasks'
+
+function deserialization(value: TodoResponse): Todo {
```

Comment:
What do you think of something clearer?

```suggestion
function deserializeTodo(value: TodoResponse): Todo {
```